Formula/leptonica.rb

```diff
@@ -1,8 +1,8 @@
 class Leptonica < Formula
   desc "Image processing and image analysis library"
   homepage "http://www.leptonica.org/"
-  url "http://www.leptonica.org/source/leptonica-1.74.1.tar.gz"
```

Comment:
I'll change it but, be aware that previous minor versions are not kept at leptonica.org so you can't install with the `1.74.1` formula because it is no longer hosted there:

```console
ep@mocha ~/Desktop $ wget http://www.leptonica.org/source/leptonica-1.74.1.tar.gz
--2017-06-27 15:02:31--  http://www.leptonica.org/source/leptonica-1.74.1.tar.gz
Resolving www.leptonica.org... 208.67.207.65
Connecting to www.leptonica.org|208.67.207.65|:80... connected.
HTTP request sent, awaiting response... 404 Not Found
2017-06-27 15:02:32 ERROR 404: Not Found.
```